src/Slim/Middleware/Session.php

```diff
@@ -43,12 +46,21 @@ public function __construct($settings = [])
             'httponly'    => false,
             'name'        => 'slim_session',
             'autorefresh' => false,
+            'handler'     => null,
         ];
         $settings = array_merge($defaults, $settings);
 
         if (is_string($lifetime = $settings['lifetime'])) {
             $settings['lifetime'] = strtotime($lifetime) - time();
         }
+        if (is_string($settings['handler'])) {            
+            $reflection = new ReflectionClass($settings['handler']);
```

Comment:
Two notes here:
- Avoid reflection when not needed. (I mean I think it's bad)
- Validation is in the `README`, users should read that always.